bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/internal/registry/Configuration.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.api.internal.registry;
+
+/**
+ * 
+ * @since 0.7
+ */
+public interface Configuration {
+
+	public static final class Empty implements Configuration {
```

Comment:
Is this needed?

Reply:
Most of service are not configurable. I expect this implementation will be the most usable one.